app/post/views/post-view-map.directive.js

```diff
@@ -29,12 +37,91 @@ function (
             query = query || PostFilters.getQueryParams($scope.filters);
 
             $scope.isLoading = true;
+
+            var postDetailPromises = [];
+
             return PostEndpoint.geojson(query).$promise.then(function (posts) {
-                map.reloadPosts(posts);
-                $scope.isLoading = false;
+                _.each(posts.features, function (feature) {
```

Comment:
This is going to be terrible for deployments that have a few posts. We only need the full post for the single post with an open popup, so loading the data for ALL of them is a waste..